.golangci.yml

```diff
@@ -0,0 +1,73 @@
+# This file contains all available configuration options
+# with their default values.
+
+# options for analysis running
+run:
```

Comment:
I remember we've discussed that we won't change linter-related things in code intentionally, but seems that base repo isn't updated for a long time, so probably we can fix linter issues.

Reply:
Some were fixed in #12, it's just that now we also add the same configuration file as in NeoGo and it finds a bit more.